docs/book/src/reference/jobs.md

```diff
@@ -26,7 +26,7 @@ Prow Presubmits:
   * GINKGO_FOCUS: `[IPv6] [PR-Informing]`, IP_FAMILY: `IPv6`
 * ✳️️ ✴️ [pull-cluster-api-e2e-full-main] `./scripts/ci-e2e.sh`
   * GINKGO_SKIP: `[PR-Blocking] [Conformance] [K8s-Upgrade] [IPv6]` (i.e. "no tags")
-* ✳️️ ✴️ [pull-cluster-api-e2e-workload-upgrade-1-24-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.24` TO: `ci/latest-1.25`
+* ✳️️ ✴️ [pull-cluster-api-e2e-workload-upgrade-1-25-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/latest-1.26`
```

Comment:
```suggestion
* ✳️️ ✴️ [pull-cluster-api-e2e-workload-upgrade-1-25-latest-main] `./scripts/ci-e2e.sh` FROM: `stable-1.25` TO: `ci/latest-1.26`
```
nit

Reply:
Should also require a corresponding change in l.84

Reply:
Thank you!